Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1393,6 +1393,13 @@ private async Task<AuthenticationResult> AuthenticateNewSessionInternal(Authenti
                 }
             }
 
+            if (user == null)
+            {
+                AuthenticationFailed?.Invoke(this, new GenericEventArgs<AuthenticationRequest>(request));
+
```

Comment:
I know you're just copy pasting, but all this extra whitespace gives me anxiety.

```suggestion
```